cfme/middleware/provider/hawkular.py

```diff
@@ -50,11 +50,13 @@ def __init__(self, name=None, hostname=None, port=None, credentials=None, key=No
             credentials = {}
         self.credentials = credentials
         self.key = key
+        self.sec_protocol = sec_protocol if sec_protocol else 'Non-SSL'
```

Comment:
@hhovsepy You will also want to add `sec_protocol=prov_config['sec_protocol']` or `sec_protocol=prov_config.get('sec_protocol')` below in `from_config` method, in case you set that value in yaml and want it loaded when instantiating providers.